ui/src/views/access-point/create-ap-preview.tsx

```diff
@@ -66,10 +98,21 @@ export const CreateAccessPointPreview: React.FC = () => {
     [prepareStatus, writeStatus, transactionStatus]
   );
 
+  const error = useMemo(
+    () => [writeStatus, transactionStatus].some((status) => status === 'error'),
+    [writeStatus, transactionStatus]
+  );
+
+  useEffect(() => {
+    if (error) {
+      AppLog.errorToast('An error occurred while minting the NFA');
+    }
+  }, [error]);

```

Comment:
This can be reduced to a single hook usage, but the better would be use the transaction error callback to show the error instead of using `useEffect`.
```suggestion
  useEffect(() => {
    const error = [writeStatus, transactionStatus].some((status) => status === 'error');
    if (error) {
      AppLog.errorToast('An error occurred while minting the NFA');
    }
  }, [writeStatus, transactionStatus]);

```